### PR TITLE
fix website for the "Factory" software

### LINKED ISSUE #91: Broken link for the Factory software
At https://oscar.computeralgebra.de/credits/, the link for Factory is ftp://www.mathematik.uni-kl.de/pub/Math/Singular/Factory, which currently doesn't work (since at least a few days).
I'm not sure who can fix that, so I will tentatively ask @hannes14 ?

### PR DESCRIPTION
**Pull request by rfourquet**
Closes #91.

### CODE REVIEW

File: _data/used_software.yml [hunk 1]
```diff
@@ -38,11 +38,10 @@
   website: http://home.imf.au.dk/jensen/software/gfan/gfan.html
 
 - name: Factory
-  website: ftp://www.mathematik.uni-kl.de/pub/Math/Singular/Factory
+  website: https://www.mathematik.uni-kl.de/ftp/pub/Math/Singular/Factory
```
Comment (fingolfin) [inline _data/used_software.yml:41]: @hannes14 is the link OK?

In any case, the link on https://www.singular.uni-kl.de/Manual/latest/sing_1.htm is also outdated (it's the same as the old one above)

Comment (fingolfin) [reply _data/used_software.yml:41]: I guess it could also link to https://www.singular.uni-kl.de/dox/html/factory_page.html

Comment (fingolfin) [reply _data/used_software.yml:41]: @hannes14 while we are at it, I made https://github.com/Singular/Sources/pull/993 (which does *not* attempt to resolve this URL, but a bunch of others on the same manual page)

**Review by hannes14 — APPROVED** [review]
the link is correct.
One may also add the link to the dox page, it contains the same README.
Difference: the first links has the tar balls, the docx link is part of Singulars doxygen pages.

**Comment by rfourquet** [comment]
I will keep only this link then, I don't think we can provide two links for one software in this page.

**Pull request merged**